common/changes/@bentley/ui-core/presentation-categories-filtering_2020-12-03-12-51.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@bentley/ui-core",
+      "comment": "",
```

Comment:
I think this should mention addition of the `tooltip` prop

Reply:
Added info about it